packages/e2e/cypress/e2e/app/createProjects.cy.ts

```diff
@@ -218,6 +218,7 @@ const defaultRowValues = [
     '11',
     '224',
     '2020-08-10',
+    '33',
```

Comment:
If we results have more columns, we may need to fix `bigqueryRowValues`, `databricksRowValues`, `snowflakeRowValues`, `trinoRowValues` as well

Reply:
You are right. Trino and Databricks are skipped ATM, I dont know why. I'm not going to update them because I think it will be misleading. 

I've updated BQ and Snowflake.